plugins/module_utils/kibana.py

```diff
@@ -167,21 +167,21 @@ def install_integration(self, integration_name, version):
         integration_install = "Cannot proceed with check_mode set to " + self.module.check_mode
       return integration_install
   
-  def check_integration(self, integration_name):
+  def check_integration(self, integration_title):
```

Comment:
Does this PR also change the `integration_name` arg in each module to `integration_title`? 

Seams like that is the case, if so, this change would break any existing playbooks out there. We should leave `integration_name` in place as is and the new arg should be named something else. This way we don't break backwards compatibility.

Reply:
I will update the playbooks.  There are only a handfull